controllers/kafkauser_controller.go

```diff
@@ -138,10 +113,30 @@ func certificateSigningRequestMapper(obj client.Object) []ctrl.Request {
 	if len(namespaceWithName) != 2 {
 		return []ctrl.Request{}
 	}
+
+	namespace := namespaceWithName[0]
+	name := namespaceWithName[1]
+
+	ctx := context.Background()
+	var kafkaUser v1alpha1.KafkaUser
+	err := m.client.Get(ctx, client.ObjectKey{Namespace: namespace, Name: name}, &kafkaUser)
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			return []ctrl.Request{}
+		}
+		m.log.Error(err, "couldn't retrieve KafkaUser", "namespace", namespace, "name", name)
+		return []ctrl.Request{}
+	}
+
+	// skip reconciling KafkaUser if owned by Schema Registry
```

Comment:
```suggestion
	// skip reconciling KafkaUser if owned by Cluster Registry
```
This seems to be a typo